### DISCUSSION TITLE
Multiline SQL query string not being highlighted via `metals` using Helix `hx`

### DISCUSSION THREAD

**Discussion by dalanmiller** [scalameta/metals#5636] [General]
I'm using `metals` with Helix (`hx`) and I'm noticing that SQL query strings are not being properly highlighted nor formatted when using `metals`. 

For example:

```
def job(args: Args): Unit = { 
  spark.sql(s"""
    SELECT *
    FROM table
    WHERE x = 'y'
  """)
}
```

Everything within the `s""" ... """` is unhighlighted. Uncertain if `metals` supports this just yet or I've potentially misconfigured something.

**Comment by kasiaMarek** [comment]
No, metals doesn't support highlighting sql queries. [Semantic token](https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#standard-token-types-and-modifiers) information sent to the editor by metals is currently built almost only based on the role the token has in the language (with very minor exceptions as using directives), specifically multiline SQL query string is a string. But you can always open a [feature request](https://github.com/scalameta/metals-feature-requests/issues).
